fluent-bit/defaults.yaml

```diff
@@ -1,10 +1,16 @@
 # -*- coding: utf-8 -*-
 # vim: ft=yaml
 fluent_bit:
-  pkg: fluentbit
+  pkg: fluent-bit
```

Comment:
The official distributed package is called `td-agent-bit`, should we use it as a default name?

Reply:
Fluent-bit it's open source, and to-agent-bit is its commercial version. I lean to opensource more.
If this formula will be able to install both of them, it probably doesn't matter.
Maybe I would opt for just short variables like "bit" and allow merge from both top pillars tree like fluent_bit or td_agent_bit. I have in general issues if variables in Salt using dashes "-" sometimes it's working sometimes don't.